web/app/components/project/index.hbs

```diff
@@ -99,16 +99,19 @@
 </div>
 
 {{#if this.jiraIsEnabled}}
-  {{! Jira }}
-  <div class="mt-10 flex w-full max-w-3xl">
-    <Project::JiraWidget
-      {{did-insert this.maybeLoadJiraInfo}}
```

Comment:
This was not getting called because of `JiraWidget`'s previous show/hide logic. The element with `attributes` wasn't being rendered so this `did-insert` went nowhere, causing the `(not @isLoading)` check to incorrectly fail.